aiida/schedulers/plugins/slurm.py

```diff
@@ -422,8 +422,14 @@ def _parse_submit_output(self, retval, stdout, stderr):
 
         Return a string with the JobID.
         """
+        from aiida.engine import CalcJob
+
         if retval != 0:
             self.logger.error(f'Error in _parse_submit_output: retval={retval}; stdout={stdout}; stderr={stderr}')
+
+            if 'Invalid account' in stderr:
+                return CalcJob.exit_codes.ERROR_SCHEDULER_INVALID_ACCOUNT
+
             raise SchedulerError(f'Error during submission, retval={retval}\nstdout={stdout}\nstderr={stderr}')
```

Comment:
I see, so an exception-based mechanism is actually in place as well...

Reply:
Yes, but this gets trapped low-down in the expontential backoff mechanism. This was exactly the direct source of the linked issue. This exception was caught and the engine kept retrying until pausing the job. The point of the exit code is that we need to get out of that, and bubble all the way up.